### PR TITLE
[IMP] payment acquirers: multi-company use

### PR DESCRIPTION
**Pull request by toaa-odoo**
Added a paragraph on users wishing to use the same payment provider across multiple companies. Forward to master.

taskid: 3095103

### CODE REVIEW

File: content/applications/finance/payment_acquirers.rst [hunk 1]
```diff
@@ -207,6 +207,19 @@ Change here the messages displayed by Odoo after a payment's confirmation or fai
 
 .. _payment_acquirers/accounting:
 
+Multi-company environment
+-------------------------
+
+If you have multiple companies and wish to use the same payment acquirer for several or all of them,
+go to :menuselection:`Configuration --> Payments: Payment Acquirers` and select the payment acquirer
```
Comment (jcs-odoo) [inline content/applications/finance/payment_acquirers.rst:214]: 1st step is to be on the "main" company, first, I think.

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
Do not fw-port it as is up to master, as you indicated, since the content isn't the same from 16 (no create button from the main payment acquirers view, no save button, and mostly, "payment acquirers" is named "payment providers"

Also, I tried this in 16.0 and 15.0, and I get an error message when trying to replicate the flow, when saving the duplicated acquirer. No error message on 14.0, though

image

Any insight on this, @AntoineVDV ?

**Comment by toaa-odoo** [comment]
> Do not fw-port it as is up to master, as you indicated, since the content isn't the same from 16 (no create button from the main payment acquirers view, no save button, and mostly, "payment acquirers" is named "payment providers"
> 
> Also, I tried this in 16.0 and 15.0, and I get an error message when trying to replicate the flow, when saving the duplicated acquirer. No error message on 14.0, though
> 
> image
> 
> Any insight on this, @AntoineVDV ?

My bad, I thought it would just conflict and I could adapt to the versions above

**Comment by AntoineVDV** [comment]
> Do not fw-port it as is up to master, as you indicated, since the content isn't the same from 16 (no create button from the main payment acquirers view, no save button, and mostly, "payment acquirers" is named "payment providers"
> 
> Also, I tried this in 16.0 and 15.0, and I get an error message when trying to replicate the flow, when saving the duplicated acquirer. No error message on 14.0, though
> 
> image
> 
> Any insight on this, @AntoineVDV ?

That's how the company switcher is designed to "work" starting Odoo 15. :roll_eyes: 
It's not related to payment. When duplicating company-dependent records, you must be careful to switch your companies at the right moment...

**Comment by toaa-odoo** [comment]
> > Do not fw-port it as is up to master, as you indicated, since the content isn't the same from 16 (no create button from the main payment acquirers view, no save button, and mostly, "payment acquirers" is named "payment providers"
> > Also, I tried this in 16.0 and 15.0, and I get an error message when trying to replicate the flow, when saving the duplicated acquirer. No error message on 14.0, though
> > image
> > Any insight on this, @AntoineVDV ?
> 
> That's how the company switcher is designed to "work" starting Odoo 15. roll_eyes It's not related to payment. When duplicating company-dependent records, you must be careful to switch your companies at the right moment...

So what does is it mean for this PR? They can only duplicate the payment provider at the very beginning ?

**Review by jcs-odoo — APPROVED** [review]
@toaa-odoo after this is merged in 14.0, you can then update first in 15.0, (if needed) and then in 16.0. Please ping me on these branches for review once you've updated them.

Cheers :)

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**